### DISCUSSION TITLE
Remove not needed default themes/plugins

### DISCUSSION THREAD

**Discussion by TheZoker** [docker-library/wordpress#627] [General]
Hi there,

I want to make my custom container as small as possible, so I tried to remove themes and plugins that I don't need:
```Dockerfile
FROM wordpress

RUN cd /var/www/html/wp-content/ \
    && rm -r themes/twentynineteen \
    && rm -r themes/twentytwenty \
    && rm -r plugins/akismet \
    && rm plugins/hello.php
```

When I try to build the container, I get the error, that the folder does not exist:
![image](https://user-images.githubusercontent.com/1368405/127162883-cbf7ff95-7226-45cb-b6b6-e3dd3d23b3e0.png)

Does anybody know why this happens?
When I don't remove these folders and inspect the build image, the folders do exists. 

Thanks!

**Comment by andreccosta** [comment]
Hi,

I believe your issue is the path in which you are removing the extras themes/plugins. You should remove them from `/usr/src/wordpress` instead.

Keep in mind they won't be removed if `/var/www/html` already exists of course (eg. your volume already exists and contains a `wp-content`). Creating the folder under `/var/www/html` is done as part of the entrypoint script only if it does no yet exists.

**Reply by TheZoker** [reply]
Very nice, that was exactly the issue! Thanks you very much, you saved me a lot of headaches!
